pkgs/os-specific/darwin/apple-source-releases/adv_cmds/xcode.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, appleDerivation, fetchurl, xcbuild, libcxx }:
+
+appleDerivation {
+  # disable pkill from build
+  patchPhase = ''
+    substituteInPlace adv_cmds.xcodeproj/project.pbxproj \
+      --replace "FD201DC214369B4200906237 /* pkill.c in Sources */," ""
```

Comment:
pkill may actually be buildable with the extra header files we've found. These are needed:

- xpc/xpc.h
- os/base_private.h
- _simple.h